scripts/git-rlog.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
```

Comment:
At your preference, it looks like you can safely add `set -euo pipefail` to these, as I don't see any area where the script wants (or needs) to plow on through error conditions.  http://redsymbol.net/articles/unofficial-bash-strict-mode/
Other than that everything looks very tidy and well-described.